Package.swift

```diff
@@ -29,7 +29,8 @@ let package = Package(
             name: "tbDEXTests",
             dependencies: ["tbDEX"],
             resources: [
-                .copy("TestVectors/ed25519")
+                .copy("TestVectors/ed25519"),
```

Comment:
Just checking, is it expected for there to be an `ed25519` folder?

Reply:
Yup, there is! https://github.com/TBD54566975/tbdex-swift/tree/main/Tests/tbDEXTests/TestVectors/ed25519